boot.php

```diff
@@ -15,9 +20,9 @@
     throw new RuntimeException('You need to pass the ROM file name (Ex: drmario.rom)');
 }
 
-$filename = 'roms/'.$argv[1];
+$filename = getcwd().'/'.$argv[1];
```

Comment:
Hum... The `getcwd().'/'` is already the default working directory, so it's unnecessary and will not make any difference but maybe, because of the PHAR archive it makes difference. I think you're right, it's better keep this way.

Perhaps, in this case, would be better if you use:
```php
getcwd().DIRECTORY_SEPARATOR.$argv[1]
```

But I don't know if it's working on Windows :stuck_out_tongue: 

What you think?